MetadataExtractor/Formats/Photoshop/PhotoshopReader.cs

```diff
@@ -71,16 +72,22 @@ public DirectoryList Extract(SequentialReader reader, int length)
                     pos += 2;
 
                     // A variable number of bytes holding a pascal string (two leading bytes for length).
-                    var descriptionLength = reader.GetByte();
+                    int descriptionLength = reader.GetByte();
```

Comment:
```suggestion
                    byte descriptionLength = reader.GetByte();
```

Reply:
I'd use the actual type here rather that widening to a signed type. It'll prevent wondering whether it may be less than zero below.